arlo_server/audit_boards.py

```diff
@@ -68,24 +69,25 @@ def assign_sampled_ballots(
         .filter_by(jurisdiction_id=jurisdiction.id)
         .join(SampledBallot.draws)
         .filter_by(round_id=round.id)
-        .order_by(SampledBallot.batch_id)  # group_by prefers a sorted list
+        .order_by(Batch.name)  # group_by prefers a sorted list
```

Comment:
what's the reasoning for switching to name vs ID?

Reply:
I explained in the PR description:

I also updated the code that splits the ballots between audit boards to key on Batch.name instead of Batch.id, which is non-deterministic and was making my tests fail due to things being in different orders on different runs. Why this didn't happen before, I don't know. Prob just got lucky.